weldx/geometry.py

```diff
@@ -2627,10 +2627,13 @@ def to_file(
         if isinstance(self._profile, VariableProfile):
             raise NotImplementedError
 
-        raster_data = self._rasterize_constant_profile(
-            profile_raster_width=profile_raster_width,
-            trace_raster_width=trace_raster_width,
-            stack=False,
+        raster_data = Q_(
+            self._rasterize_constant_profile(
+                profile_raster_width=profile_raster_width,
+                trace_raster_width=trace_raster_width,
+                stack=False,
+            ),
+            _DEFAULT_LEN_UNIT,
         )
```

Comment:
in codeblocks like these the added indent from `Q_()` sometimes makes it a little harder to read imo
In that case the `* _DEFAULT_LEN_UNIT` syntax might make it more readable (it is also better to see in the diffs)